build.sbt

```diff
@@ -26,42 +26,22 @@ def SbtPluginPlugins  = Sonatype
 
 lazy val `sbt-relay-compiler` = project
   .in(file("sbt-plugin"))
-  .enablePlugins(SbtPluginPlugins)
-  .enablePlugins(SbtPlugin)
+  .enablePlugins(SbtPlugin, SbtPluginPlugins)
   .settings(commonSettings ++ mavenSettings)
   .settings(
     sbtPlugin := true,
+    // TODO: Why are these here?
     addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % Version.Scalajs),
```

Comment:
I actually don't know if these are necessary... it would have been because we are depending on scalajs tasks or something in the plugin.